lib/color_guard/store/stack.rb

```diff
@@ -19,6 +19,10 @@ def find(feature_name)
         end
       end
 
+      def all
+        @stores.map { |store| store.all }
+      end
+
```

Comment:
We could have `#all` return a hash so that it has the same interface as the stores.

And then we can have a `#all_features` (or any other method name) that will return an array of hashes just for the UIs sake.